examples/gcp-cloudrun/infra/project.scala

```diff
@@ -1,6 +1,6 @@
 //> using scala "3.3.1"
 //> using options -Werror -Wunused:all -Wvalue-discard -Wnonunit-statement
-//> using plugin "org.virtuslab::besom-compiler-plugin:0.2.2"
+///> using plugin "org.virtuslab::besom-compiler-plugin:0.2.2"
```

Comment:
if we are disabling it why not remove instead of commenting out?

Reply:
I have it disabled only because scala-cli has a known bug when working with bloop and metals where metals get wrecked by compiler plugins and some compiler options (like disable autotupling). My mistake that it made it to the commit.